build-image/first-boot.bash

```diff
@@ -18,7 +18,9 @@ fail_inprogress() {
   exit 1
 }
 
-echo "$(timestamp) [openHABian] Starting the openHABian initial setup."
+###### start ######
+sleep 5
```

Comment:
Why add a delay before start?

Reply:
systemd might start more services, this improves the output

Reply:
OK